packages/auth-compat/index.node.ts

```diff
@@ -23,15 +23,10 @@
  */
 export * from './index';
 import { FetchProvider } from '@firebase/auth/internal';
-import {
-  fetch as undiciFetch,
-  Headers as undiciHeaders,
-  Response as undiciResponse
-} from 'undici';
 import './index';
 
 FetchProvider.initialize(
-  undiciFetch as unknown as typeof fetch,
-  undiciHeaders as unknown as typeof Headers,
-  undiciResponse as unknown as typeof Response
+  fetch as unknown as typeof fetch,
```

Comment:
As Christina pointed out in the other comment, we also don't need casting here, correct?

Reply:
Done! Thanks!